impala/hiveserver2.py

```diff
@@ -126,7 +127,11 @@ def cursor(self, user=None, configuration=None, convert_types=True,
 
         log.debug('.cursor(): getting new session_handle')
 
-        session = self.service.open_session(user, configuration)
+        session_configuration = {}
```

Comment:
The comments about configuration could be updated to reflect that it overrides connection level cursor_configuration